### DISCUSSION TITLE
Serverpod generate is not adding the tables to postgres

### DISCUSSION THREAD

**Discussion by ajay-k07** [serverpod/serverpod#491] [Q&A] [answered]
I have added the Auth module for server ref:- [https://docs.serverpod.dev/concepts/modules](https://docs.serverpod.dev/concepts/modules)

![Screenshot 2022-12-16 at 2 59 47 PM](https://user-images.githubusercontent.com/37687348/208067550-ac01c771-5566-4691-abf5-43994cba93d0.png)

![Screenshot 2022-12-16 at 3 07 30 PM](https://user-images.githubusercontent.com/37687348/208069040-50f38b55-6e7f-4f99-9871-900510b2c2a8.png)

After Running 
`dart pub get                                        
serverpod generate`

i dont see any tables related code gets generated in generated/tables.pgsql. do i need to add that manually ?

![Screenshot 2022-12-16 at 3 08 19 PM](https://user-images.githubusercontent.com/37687348/208069236-6d85da43-f981-4c98-a8f9-7f57d16fbe0b.png)

After adding the tables manually from pg admin its working. So my doubt is do we need to add the tables or serverpod will take care of that 

if its currently done as manual work is there any chance for us to automate that

**Comment by vlidholt** [comment, marked as answer]
You currently have to manually add the tables. We hope to fix this in version 1.1.

**Reply by II11II** [reply]
Is there ticket(task) for this?

**Reply by yahu1031** [reply]
@ajay-k07 Checkout the first para last statement please in [Database setup](https://docs.serverpod.dev/concepts/modules#database-setup)

**Comment by II11II** [comment]
@vlidholt should `serverpod generate` command adds new tables to the Database or will there be another command?

**Reply by vlidholt** [reply]
It will not happen through the `serverpod generate` command. Instead, most likely when starting/restarting your serverpod.
